### PR TITLE
cerebro - adding support for host auth values headers

### PR DESCRIPTION
**Pull request by webdz9r**
According to the source code it [supports http auth headers ](https://github.com/lmenezes/cerebro/blob/ccb60448b030ead90cb7309221e296c8da920175/conf/application.conf#L80) in the host definition. While yes you can create a custom application.conf file via a secret I thought I'd be nice to supported in the helm chart

additionally adding header-whitelist value as well per the source supports it

### CODE REVIEW

File: charts/cerebro/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: cerebro
-version: 2.0.3
+version: 2.0.4
```
Comment (maxime1907) [inline charts/cerebro/Chart.yaml:2]: its a feature so 2.1.0

File: charts/cerebro/templates/secret.yaml [hunk 2]
```diff
@@ -74,6 +74,14 @@ stringData:
       {
         host = {{ $element.host | quote }}
         name = {{ $element.name | quote }}
+        {{- if $element.headersWhitelist }}
+        headers-whitelist = {{ $element.headersWhitelist }} {{- end}}
```
Comment (maxime1907) [inline charts/cerebro/templates/secret.yaml:78]: could you put `{{- end }}` after `headers-whitelist`

**Comment by bgrande12** [comment]
@webdz9r bump -- can you make the changes in your fork?

**Comment by maxime1907** [comment]
Closing this since i have merged the changes here:
- https://github.com/wiremind/wiremind-helm-charts/pull/522

Thanks for your contribution!

**Pull request closed without merging**